lib/public/utilities/ObservableData.js

```diff
@@ -43,4 +43,13 @@ export class ObservableData extends Observable {
         this._current = value;
         this.notify();
     }
+
+    /**
+     * Silently set value of observable data
+     * @param {*} value new value
+     * @return {void}
+     */
+    setSilently(value) {
+        this._current = value;
+    }
```

Comment:
I think that if you need that, it means that there is a misconfiguration in the observation chain